widget/templates/_manifest.json

```diff
@@ -3,9 +3,9 @@
   "label": "<%= widgetTitle %>",
   "2D": <%= is2d %>,
   "3D": <%= is3d %>,
-  "platform": "HTML",
-  "version": "0.0.1",
-  "wabVersion": "1.4",
+  "platform": "<%= platform %>",
+  "version": "<%= version %>",
```

Comment:
I beleive `version` refers to the version of the widget itself. It should _always_ be "0.0.1" for the purposes of this generator. If you see OOTB WAB widgets that show "2.0beta" for this setting, I'm guessing that's b/c it's the v2.0beta of that widget.